algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/treatment/HasHadSystemicTreatmentInAdvancedOrMetastaticSettingTest.kt

```diff
@@ -17,61 +19,95 @@ class HasHadSystemicTreatmentInAdvancedOrMetastaticSettingTest {
     private val referenceDate = LocalDate.of(2024, 11, 26)
     private val recentDate = referenceDate.minusMonths(1)
     private val nonRecentDate = recentDate.minusMonths(7)
-    private val function = HasHadSystemicTreatmentInAdvancedOrMetastaticSetting(referenceDate)
+    private val advancedOrMetastaticFunction = HasHadSystemicTreatmentInAdvancedOrMetastaticSetting(referenceDate, metastaticOnly = false)
```

Comment:
if we did pass the intents in the constructor you wouldn't need to have all these "duplicated" tests.